src/script/target/webpack/getConfiguration.js

```diff
@@ -0,0 +1,18 @@
+const overrideConfiguration = require('./overrideConfiguration')
+const getRaw = require('./getRaw')
+const setEnv = require('./setEnv')
+
+module.exports = async (state, callbacks) => {
```

Comment:
Why do you make a generic `callbacks` object instead of the waiting callback `getProjectPath` ?

More globally, `projectPath` should be initialize in the `start.js`, because it's a required condition before starting anything else. _Same remark for `webpackConfigPath`_